JustSaying/Fluent/TopicSubscriptionBuilder`1.cs

```diff
@@ -100,13 +100,14 @@ public TopicSubscriptionBuilder<T> WithReadConfiguration(Action<SqsReadConfigura
         void ISubscriptionBuilder<T>.Configure(JustSayingFluently bus, IHandlerResolver resolver)
         {
             var topic = bus.WithSqsTopicSubscriber()
-                           .IntoQueue(TopicName)
-                           .WithMessageHandler<T>(resolver);
+                           .IntoQueue(TopicName);
 
             if (ConfigureReads != null)
             {
                 topic.ConfigureSubscriptionWith(ConfigureReads);
             }
+
+            topic.WithMessageHandler<T>(resolver);
```

Comment:
I see the fix!

does the file name here really end with `1 ?

Reply:
There are several files where the name ends in "`1.cs" 
Is that intentional? Change in a separate PR?

Reply:
I've seen it in a few other repos and I like it cause you don't have to think about naming the file vs the type, sometimes there's a non-generic type that would clash.

Reply:
`IEnumerable.cs` vs. ``IEnumerable`1.cs`` for instance.